docs/engineering/android-app/configuring/config-types/application.mdx

```diff
@@ -23,12 +23,39 @@ There can only be one instance of application configuration for the entire appli
     ],
     "useDarkTheme": false,
     "syncInterval": 15,
-    "syncStrategy": [
-        "Location",
-        "Organization",
-        "CareTeam",
-        "Practitioner"
+    "syncStrategy": "Location",
```

Comment:
@dubdabasoduba @ndegwamartin Confirm if this should be String or JsonArray? Do we support multiple sync strategies?

Reply:
I don't think we should/do. For the case of the old OpenSRP where we updated the sync filter programatically to fetch depending on if supervisor - we now cater for that use case by using a different app-id for the different role where they can have their own strategy.

@dubdabasoduba can add on this

Reply:
I agree with @ndegwamartin. The combination of App Id and Sync Strategies caters for this.  

We might end up restructuring how we sync and tag resources but we can figure out the exact changes required when requirement gathering and scoping are done.